site/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "hnpwa-site"
```

Comment:
Yep! Same IAM permissions as GCP. I can easily add anyone or we can create an admin account as the owner.